src/intents.cs

```diff
@@ -925,6 +1010,8 @@ public enum INWorkoutLocationType : long {
 	[iOS (10, 0)]
 	[Mac (10, 12, 0, PlatformArchitecture.Arch64)]
 	[Watch (3, 2)]
+	[Mac (11,0)]
```

Comment:
So we have [Mac (10, 12, 0, PlatformArchitecture.Arch64)] and [Mac (11,0)], is that a problem?